UPGRADING.rst

```diff
@@ -66,3 +63,17 @@ Examples:
 +-------------+---------------+-----------+-------------+
 | ``foo-bar`` | ``-``         | 2         | 2           |
 +-------------+---------------+-----------+-------------+
+
+Graylog REST API
+================
+
+Streams API
+-----------
+
+Due to the introduction of index sets, the payload for creating, updating and cloning of streams now requires the ``index_set_id`` field. The value for this needs to be the ID of an existing index set.
+
+Affected endpoints:
+
+* ``POST /streams``
+* ``PUT  /streams/{streamId}``
```

Comment:
![image](https://cloud.githubusercontent.com/assets/461/21858384/298eadd8-d829-11e6-8368-b66a46e94f89.png)

We are not sending the complete resource when updating a stream. This basically violates the PUT HTTP verb, but well. History. :smiley: 

That's why it needs to be nullable.